spec/features/sign_up_spec.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../web_helpers'
```

Comment:
We could require it in spec_helpers to avoid repetition in sign_in_spec.rb and sign_up_spec.rb